features/postmessage.yml

```
@@ -0,0 +1,16 @@
+name: PostMessage
+description: "The `postMessage()` method safely enables cross-origin communication for windows and workers, including popups and iframes."
```

Suggestions here: putting this into a more active voice (see: [enables](https://github.com/web-platform-dx/web-features/blob/main/docs/guidelines.md#enables)) and adding an _also known as_.

Also (_gravelly action move voice_), no one is safe.

```suggestion
description: "The `postMessage()` global method sends cross-origin messages to windows and workers, including popups and iframes. Also known as cross-document messaging."
```